yt/yt/server/master/cell_server/tamed_cell_manager.cpp

```diff
@@ -1093,7 +1093,7 @@ class TTamedCellManager
         if (!cellBundle) {
             THROW_ERROR_EXCEPTION(
                 NYTree::EErrorCode::ResolveError,
-                "No such %Qlv cell bundle %Qlv",
+                "No such %lv cell bundle %Qlv",
```

Comment:
Why Qlv for name?

Reply:
I think there is no reason, it should be %Qv. WIll fix in a separate PR.

Reply:
https://github.com/ytsaurus/ytsaurus/pull/937